dao/src/main/java/se/fortnox/reactivewizard/db/DbProxy.java

```diff
@@ -1,97 +1,57 @@
 package se.fortnox.reactivewizard.db;
 
-import com.fasterxml.jackson.core.type.TypeReference;
 import jakarta.inject.Inject;
 import jakarta.inject.Singleton;
-import org.reactivestreams.Publisher;
-import reactor.core.publisher.Flux;
-import reactor.core.publisher.Mono;
 import reactor.core.scheduler.Scheduler;
 import reactor.core.scheduler.Schedulers;
 import se.fortnox.reactivewizard.db.config.DatabaseConfig;
 import se.fortnox.reactivewizard.db.paging.PagingOutput;
-import se.fortnox.reactivewizard.db.statement.DbStatementFactory;
 import se.fortnox.reactivewizard.db.statement.DbStatementFactoryFactory;
-import se.fortnox.reactivewizard.db.transactions.ConnectionScheduler;
-import se.fortnox.reactivewizard.json.JsonSerializerFactory;
 import se.fortnox.reactivewizard.metrics.Metrics;
 import se.fortnox.reactivewizard.util.DebugUtil;
 import se.fortnox.reactivewizard.util.ReflectionUtil;
 
-import javax.annotation.Nullable;
 import java.lang.reflect.InvocationHandler;
 import java.lang.reflect.Method;
 import java.lang.reflect.Proxy;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Function;
 
 import static java.text.MessageFormat.format;
 
 @Singleton
 public class DbProxy implements InvocationHandler {
 
-    private static final TypeReference<Object[]> OBJECT_ARRAY_TYPE_REFERENCE = new TypeReference<>() {
-    };
     private final DbStatementFactoryFactory dbStatementFactoryFactory;
-    private final Scheduler scheduler;
-    protected final Map<Method, ReactiveStatementFactory> statementFactories;
-    private final ConnectionScheduler connectionScheduler;
-    protected final Function<Object[], String> paramSerializer;
-    private final DatabaseConfig databaseConfig;
+    protected final Map<Method, DaoMethodHandler> handlers;
+    private final ReactiveStatementFactory reactiveStatementFactory;
 
     @Inject
-    public DbProxy(DatabaseConfig databaseConfig,
```

Comment:
I'm perplexed about the removal of the JsonSerializerFactory, was it never used anywhere?